lib/inspec/base_cli.rb

```diff
@@ -64,6 +64,8 @@ def self.exec_options
       option :attrs, type: :array,
         desc: 'Load attributes file (experimental)'
       option :cache, type: :string,
+        desc: '[DEPRECATION] Please use --vendor-cache - this will be removed in InSpec 2.0'
```

Comment:
Can we change this to `[DEPRECATED]`? It's cleared to the users that it's deprecated, vs. us declaring that it's a deprecation. A nitpick, I know, but grammar matters :)